Automattic/wp-calypso
# i18n: Exclude build-languages-if-enabled script execution from desktop builds

#### Changes proposed in this Pull Request

This PR will exclude `build-languages-if-enabled` execution from desktop builds `yarn run build-client-if-desktop`, because language files and translation chunks are currently not being used for desktop builds. 

`build-languages-if-enabled` is now attached to `build-client-if-prod` script and will only be used for production builds.

#### Testing instructions

1. Run `CALYPSO_ENV=production BUILD_TRANSLATION_CHUNKS=true yarn run build` and confirm that `build-languages-if-enabled` script is being executed.
2. Run `CALYPSO_ENV=desktop BUILD_TRANSLATION_CHUNKS=true yarn run build` and confirm that `build-languages-if-enabled` script is NOT being executed.
